examples/core/instancing/app.js

```diff
@@ -47,21 +51,13 @@ const animationLoop = new AnimationLoop({
     });
 
     const pickInfo = pickModels(gl, {models: [cube], position: pickPosition, framebuffer});
-    // console.log(pickInfo ? pickInfo.color : 'picked background');
     cube.updateModuleSettings({
-      pickingSelectedColor: pickInfo && pickInfo.color
+      pickingSelectedColor: pickInfo && pickInfo.color,
+      pickingValid: pickInfo !== null
```

Comment:
Do we really need this separate variable? Isn't `pickingSelectedColor` set to `null` when `pickInfo` is `null`?